spec/domain/accounts/users_spec.exs

```diff
@@ -0,0 +1,161 @@
+defmodule Squadster.Domain.Accounts.UsersSpec do
+  use ESpec.Phoenix, async: true
+  use ESpec.Phoenix.Extend, :model
+
+  alias Squadster.Accounts.User
+
+  describe "changeset" do
+    context "when params are valid" do
+      it "is valid" do
+        %{valid?: is_valid} = %{first_name: "John", last_name: "Galt"} |> User.changeset
+        expect is_valid |> to(be true)
+      end
+    end
+
+    context "when first_name is not set" do
+      it "is not valid" do
+        %{valid?: is_valid} = %{last_name: "Galt"} |> User.changeset
+        expect is_valid |> to(be false)
+      end
+    end
+
+    context "when last_name is not set" do
+      it "is not valid" do
+        %{valid?: is_valid} = %{first_name: "John"} |> User.changeset
+        expect is_valid |> to(be false)
+      end
+    end
+
+    context "when mobile_phone contains illegal symbols" do
+      it "is not valid" do
+        %{valid?: is_valid} = %{first_name: "John", last_name: "Galt", mobile_phone: "&123"} |> User.changeset
+        expect is_valid |> to(be false)
+      end
+    end
+  end
+
+  describe "auth_changeset" do
+    context "when params are valid" do
+      it "is valid" do
+        %{valid?: is_valid} = %{uid: "123", first_name: "John", last_name: "Galt"} |> User.changeset
+        expect is_valid |> to(be true)
+      end
+    end
+
+    context "when first_name is not set" do
+      it "is not valid" do
+        %{valid?: is_valid} = %{uid: "123", last_name: "Galt"} |> User.auth_changeset
+        expect is_valid |> to(be false)
+      end
+    end
+
+    context "when last_name is not set" do
+      it "is not valid" do
+        %{valid?: is_valid} = %{uid: "123", first_name: "John"} |> User.auth_changeset
+        expect is_valid |> to(be false)
+      end
+    end
+
+    context "when uid is not set" do
+      it "is not valid" do
+        %{valid?: is_valid} = %{first_name: "John", last_name: "Galt"} |> User.auth_changeset
+        expect is_valid |> to(be false)
+      end
+    end
+  end
+
+  describe "auth functions" do
```

Comment:
Maybe make sence to move it to `fixture` of `factory`

Reply:
Good catch, done